src/pudl/transform/eia.py

```diff
@@ -319,12 +318,6 @@ def _compile_all_entity_records(entity, eia_transformed_dfs):
                 df["table"] = table_name
                 dfs.append(df)
 
-                # remove the static columns, except the explicitly non-dropped cols
-                transformed_df = transformed_df.drop(
-                    columns=[c for c in cols if c not in keep_cols]
-                )
-                eia_transformed_dfs[table_name] = transformed_df
-
```

Comment:
No longer required, since all of the entity harvesting steps happen independent of each other, and column dropping is done on all these dataframes after the fact using `enforce_schema()`.

Reply:
Much cleaner this way! 🧹